### ISSUE TITLE
TemplateNotFound: salt/provision.sh.j2 - failed build

### ISSUE THREAD

**Issue by asettle** [SUSE/sesdev#423]
Attempting to build a ses7 cluster using sesdev on openSUSE Leap 15.1 using these instructions: https://github.com/SUSE/sesdev#install-sesdev-on-opensuse-or-suse-linux-enterprise

After a successful deploy of sesdev (running `sesdev list` or `sesdev --help` has a successful return), attempting to run: `sesdev create ses7` fails with the following traceback:

`sesdev create ses7
Traceback (most recent call last):
  File "/usr/bin/sesdev", line 11, in <module>
    load_entry_point('sesdev==1.6.0+1595760860.gc353ea1', 'console_scripts', 'sesdev')()
  File "/usr/lib/python3.6/site-packages/sesdev/__init__.py", line 38, in sesdev_main
    cli(prog_name='sesdev')
  File "/usr/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/sesdev/__init__.py", line 876, in ses7
    _create_command(deployment_id, deploy, settings_dict)
  File "/usr/lib/python3.6/site-packages/sesdev/__init__.py", line 737, in _create_command
    dep = Deployment.create(deployment_id, settings)
  File "/usr/lib/python3.6/site-packages/seslib/deployment.py", line 1382, in create
    dep.save()
  File "/usr/lib/python3.6/site-packages/seslib/deployment.py", line 492, in save
    scripts = self._generate_vagrantfile()
  File "/usr/lib/python3.6/site-packages/seslib/deployment.py", line 485, in _generate_vagrantfile
    scripts['provision_{}.sh'.format(node.name)] = template.render(**context_cpy)
  File "/usr/lib/python3.6/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/usr/lib/python3.6/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/usr/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/usr/lib/python3.6/site-packages/seslib/templates/provision.sh.j2", line 38, in top-level template code
    {% include "salt/provision.sh.j2" %}
  File "/usr/lib/python3.6/site-packages/jinja2/loaders.py", line 235, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: salt/provision.sh.j2`

After looking into it further, the folder `salt/provision.sh.j2` does not exist. If you review `/usr/lib/python3.6/site-packages/seslib/templates/provision.sh.j2` closely, you'll see the following:

```# deploy
{% if deploy_salt or suma %}
{% include "salt/provision.sh.j2" %}
{% elif version == 'caasp4' %}
{% include "caasp/provision.sh.j2" %}
{% elif version == 'makecheck' %}
{% include "makecheck/provision.sh.j2" %}
{% endif %}
```
Reviewing the files, you'll see that the `caasp` folder exists, but the `salt` folder does not:

```sesdev-dashboard:~ # ls -la /usr/lib/python3.6/site-packages/seslib/templates/
total 32
drwxr-xr-x 4 root root 4096 Aug 11 11:10 .
drwxr-xr-x 4 root root 4096 Aug 11 10:31 ..
-rw-r--r-- 1 root root 1629 Jul 26 10:54 Vagrantfile.j2
drwxr-xr-x 2 root root 4096 Aug 11 10:31 caasp
drwxr-xr-x 3 root root 4096 Aug 11 10:31 engine
-rw-r--r-- 1 root root 1302 Aug 11 11:07 provision.sh.j2
-rw-r--r-- 1 root root 3351 Jul 26 10:54 sync_clocks.j2
-rw-r--r-- 1 root root 3981 Jul 26 10:54 zypper.j2
```

Either something has gone weirdly wrong with this deploy, or something is amiss. 

I was able to successfully deploy if I edited the `usr/lib/python3.6/site-packages/seslib/templates/provision.sh.j2` to change `{% include "salt/provision.sh.j2" %}` to say `{% include "caasp/provision.sh.j2" %}` - however this is clearly not what we want.

Any help appreciated!

**Comment by smithfarm** [comment]
When you say "a successful deploy of sesdev", are you referring to deployment of sesdev from (a) source code or (b) RPM package?

**Comment by smithfarm** [comment]
Ah, you are installing from RPM. I see the issue and will have a fix out shortly. Thanks for reporting and sorry for the inconvenience!

In the meantime, you can work around this issue by installing sesdev from source code (according to the instructions in the README).

**Comment by smithfarm** [comment]
Thanks for the well-written bug report. It enabled me to find the problem quickly. Fixed by #424
